docs/docs/index.md

```diff
@@ -10,15 +10,14 @@ Welcome to the Browsertrix Crawler official documentation.
 
 Browsertrix Crawler is a simplified browser-based high-fidelity crawling system, designed to run a complex, customizable browser-based crawl in a single Docker container. Browsertrix Crawler uses [Puppeteer](https://github.com/puppeteer/puppeteer) to control one or more [Brave Browser](https://brave.com/) browser windows in parallel. Data is captured through the [Chrome Devtools Protocol (CDP)](https://chromedevtools.github.io/devtools-protocol/) in the browser.
 
+Browsertrix Crawler is a command line application responsible for the core features of [Browsertrix](https://browsertrix.com), Webrecorder's cloud-based web archiving service. You can find the documentation for Browsertrix — the cloud platform — [here](https://docs.browsertrix.cloud)!
```

Comment:
```suggestion
Browsertrix Crawler is a command line application responsible for the core features of [Browsertrix](https://browsertrix.com), Webrecorder's cloud-based web archiving service. See the [Browsertrix documentation] for more information about Browsertrix, the cloud platform.
```

Generally good to avoid link text like "here", maybe this is a way of doing that?

This could also possibly be a note, though I don't have a strong feeling about that?

Reply:
You know, I do this a lot and should probably stop... Example [here](https://wilkinson.graphics/) ;P

Will reword!